src/main/java/org/folio/inventory/dataimport/handlers/matching/AbstractMatchEventHandler.java

```diff
@@ -86,4 +147,5 @@ public boolean isEligible(DataImportEventPayload dataImportEventPayload) {
 
   protected abstract String getNotMatchedEventType();
 
+  protected abstract boolean isConsortiumAvailable();
```

Comment:
```suggestion
  protected abstract boolean isConsortiumActionAvailable();
```
let's make it a be more clear